### PR TITLE
RHS ARF Compat - Fix RPK74M wrong ammo bug

### PR DESCRIPTION
**Pull request by Dabako**
**When merged this pull request will:**
- This patch will fix the bug #8221
- Add of RPK74M in optionals/compat_rhs_afrf3/CfgWeapons.hpp

I have tested it locally and it worked as intended

### CODE REVIEW

**Comment by reyhard** [comment]
class rhs_weap_rpk74: rhs_weap_pkp { 
is still there so it will be still broken, just slightly less

**Comment by Dabako** [comment]
I changed that in the second commit, I just checked it and it should not be there

File: optionals/compat_rhs_afrf3/CfgWeapons.hpp [hunk 1]
```diff
@@ -57,6 +57,10 @@ class CfgWeapons {
     class rhs_weap_svds_npz: rhs_weap_svds {
         ACE_RailHeightAboveBore = 4.3348;
     };
+    class rhs_weap_rpk74_base;
+    class rhs_weap_rpk74m : rhs_weap_rpk74_base {
```
Comment (PabstMirror) [inline optionals/compat_rhs_afrf3/CfgWeapons.hpp:61]: After looking at their configs I think `class rhs_weap_rpk74_base: rhs_weap_rpk_base` would be a better choice
and probably keep the `ACE_barrelTwist = 195.072;` config from before as well

**Review by PabstMirror — CHANGES_REQUESTED** [review]

File: optionals/compat_rhs_afrf3/CfgWeapons.hpp (hunk 1 above)
Comment (Dabako) [reply optionals/compat_rhs_afrf3/CfgWeapons.hpp]: Changed in newest commit, tested it and it works for me too

**Review by PabstMirror — APPROVED** [review]

**Pull request merged**